### DISCUSSION TITLE
It's not working

### DISCUSSION THREAD

**Discussion by Erikcb91** [DELUUXE/DiscordChroma#31] [General]
Hi, is the app still working? can't make it to work, it just says it has encountered an error

![image](https://user-images.githubusercontent.com/75399666/229604552-e50664cc-d6ed-42ea-8fef-089436571bab.png)

Sorry if this is not the place to ask it, i don't know where i should ask.
Thanks!

**Comment by DELUUXE** [comment]
Hi @Erikcb91, could you share the log file after a crash like that happens?
it should be located in ``%appdata%/DiscordChroma/log.txt``
This log file can contain usernames and channel names of messages you received, so you may want to remove those before posting.
(this is a public forum after all)

**Comment by Erikcb91** [comment]
This is the logs i get, by the way, what are the animations so far? Can't seem to find a list anywhere


[log.txt](https://github.com/DELUUXE/DiscordChroma/files/11148200/log.txt)

**Reply by DELUUXE** [reply]
Currently there is only 1 animation, a blue/purple (discord logo) color when you get a message as far as i remember.
Though i don't really maintain this project anymore at the moment, so there won't be any new ones most likely.

As for the error you're getting, it seems that there is an issue with detecting whether discord is running or not.
Could you provide some more details about your operating system etc.?

**Reply by Erikcb91** [reply]
I see

![image](https://user-images.githubusercontent.com/75399666/230152587-f8af26cf-3043-43af-a75e-6120e771932a.png)

This is my windows version

**Reply by DELUUXE** [reply]
Sorry for the huge delay.
However, this issue seems to be because you are not an administrator on that pc?
I did not take in to account the method i used for detecting whether Discord is running requires an admin account.
And i am currently not sure when i have time to find an alternative method.

**Reply by Erikcb91** [reply]
It's an administrator account, so that shouldnt be the issue, don't worry, i'm in no hurries.

**Reply by DELUUXE** [reply]
Hmm, the information i could find about the error you posted mentioned not having admin rights.
Either way a different method of detecting if discord is running might be better, I'll give you an update when i have more to share.

**Reply by Erikcb91** [reply]
awesome, thanks!
